sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -89,53 +118,26 @@ private boolean matchSpringBootVersionFromManifest(String s) throws FileNotFound
             LOGGER.info("Cannot check Boot version from manifest");
             throw new FileNotFoundException("Spring Boot version not found");
         } else {
-            return version.startsWith(stripWildCardFromVersion(s));
+            return version.startsWith(stripWildCardFromVersion(acceptedVersion));
         }
     }
 
     private static String stripWildCardFromVersion(String version) {
         return version.endsWith(".x") ? version.substring(0, version.indexOf(".x")) : version;
     }
 
-    private CompatibilityPredicate is2_6() {
-        return new CompatibilityPredicate() {
-            public String toString() {
-                return "Predicate for Boot 2.6";
+    @Override
+    public boolean test(String fullyQuallifiedClassName) {
+        try {
+            if (null == fullyQuallifiedClassName) {
```

Comment:
`fullyQuallifiedClassName == null`

we don't do that, put the null first

Reply:
Ok, will follow this style.